notebooks/D2-entitlement-regression.ipynb

```diff
@@ -103,13 +92,21 @@
     "    verbose=True,\n",
     "    big_case_threshold=20,\n",
     "    return_big_cases=False,\n",
-    "    aggregate_years=True,\n",
```

Comment:
@tiffanychu90 This is likely the issue you were seeing. This is a new(ish) parameter that determines whether to return counts broken down by year or aggregated.